spec/integration/_dev/build/build.spec.yml

```diff
@@ -20,7 +20,17 @@ spec:
               type: string
               description: Source reference
               pattern: '^git@.+'
+            import_common_dynamic_mappings:
+              type: boolean
+              description: >
+                Whether or not import common used dynamic templates and properties into the package
+              default: false
           required:
             - reference
   required:
-  - dependencies
\ No newline at end of file
+  - dependencies
+versions:
+  - before: 2.3.0
+    patch:
+      - op: remove
+        path: "/properties/dependencies/properties/ecs/properties/import_common_dynamic_mappings"
```

Comment:
:+1: this will encourage users of this to update the format of their packages :slightly_smiling_face: